android/build.gradle

```diff
@@ -1,11 +1,74 @@
-// On top of your file import a JSON parser
 import groovy.json.JsonSlurper
 
+import java.nio.file.FileVisitResult
+import java.nio.file.Files
+import java.nio.file.Path
+import java.nio.file.SimpleFileVisitor
+import java.nio.file.attribute.BasicFileAttributes
+
+String getPackageJsonPath() {
+    return findProperty("APPSFLYER_PACKAGE_JSON") ?: "$rootDir/../node_modules/appsflyer-capacitor-plugin/package.json"
```

Comment:
Where and how can I define the `APPSFLYER_PACKAGE_JSON` variable?

Reply:
Update: I just found out I can place `ext { APPSFLYER_PACKAGE_JSON = '<path>' }` into a `variables.gradle` file that is used via `apply from: "variables.gradle"` in our `build.gradle`.

Reply:
Why don't you do "$buildFile/../../package.json" instead of "$rootDir/../node_modules/appsflyer-capacitor-plugin/package.json" ? Then you don't need workarounds in project that integrates with your SDK.